roles/openshift_logging/README.md

```diff
@@ -222,3 +222,27 @@ The corresponding openshift\_logging\_mux\_* parameters are below.
 - `openshift_logging_mux_remote_syslog_tag_key`: If string specified, use this field from the record to set the key field on the syslog message
 - `openshift_logging_mux_remote_syslog_use_record`: Set `true` to use the severity and facility from the record, defaults to `false`
 - `openshift_logging_mux_remote_syslog_payload_key`: If string is specified, use this field from the record as the payload on the syslog message
+
+Image update procedure
+----------------------
+
+An upgrade of the logging stack from older version to newer should always be performed by calling appropriate ansible playbook and setting required ansible variables in your inventory as documented in https://docs.openshift.org/.
+
+There could be a situation where user may want to update only the logging images and not the entire logging stack setup.
+
+In order to do so, recommended procedure is:
```

Comment:
@wozniakjan, where do we provide the ansible playbook to implement the recommended procedure?

Reply:
We don't have any, it was never requested.

Reply:
@portante We should eventually update the ansible roles to check for and pull the latest images of the specific version as part of an installation. Its a tech debt item along with some others I have in my mind...